dev-client/components/Login.tsx

```diff
@@ -0,0 +1,36 @@
+import {Button, VStack} from 'native-base';
+import {useCallback} from 'react';
+import {authorize} from 'react-native-app-auth';
+
+import Config from 'react-native-config';
+import {exchangeToken} from '../src/auth';
+import {useLoginDispatch} from '../src/context/LoginContext';
+
+// https://github.com/FormidableLabs/react-native-app-auth/blob/main/docs/config-examples/google.md
+const googleConfig = {
+  issuer: 'https://accounts.google.com',
+  clientId: `${Config.GOOGLE_OAUTH_APP_GUID}.apps.googleusercontent.com`,
+  redirectUrl: 'com.devclient:/oauth2redirect',
+  scopes: ['openid', 'profile', 'email'],
+};
+
+export default function LoginView(): JSX.Element {
+  const loginDispatch = useLoginDispatch();
+  const onPress = useCallback(() => {
+    authorize(googleConfig)
+      .then(async result => {
+        let {atoken, rtoken, email, firstName, lastName} = await exchangeToken(
+          result.idToken,
+          'google',
+        );
+        loginDispatch({type: 'login', name: firstName});
+      })
+      .catch(e => console.error(e));
+  }, [googleConfig, authorize]);
```

Comment:
nit: i think `googleConfig` and `authorize` will never change, can we omit them from the keys? or does linter complain even tho they're consts?